tdp/cli/commands/deploy.py

```diff
@@ -113,14 +113,16 @@ def deploy(
             raise click.ClickException(str(e)) from e
         deployment_iterator = deployment_runner.run(deployment_plan)
         if dry:
-            for operation in deployment_iterator:
+            for _ in deployment_iterator:
                 pass
         else:
             session.add(deployment_iterator.log)
             # insert pending deployment log
             session.commit()
-            for operation in deployment_iterator:
-                session.add(operation)
+            for operation_log, service_log in deployment_iterator:
```

Comment:
It is a `service_log` ? Not a `service_component_log` ?

Reply:
Fixed